packages/compass-connect/test/renderer/stores/index.spec.js

```diff
@@ -1052,7 +1052,7 @@ describe('Store', () => {
 
             it('sets the driverUrl', (done) => {
               const driverUrl =
-                'mongodb://server.example.com:27017/?readPreference=primary&directConnection=true&ssl=false';
+                'mongodb://server.example.com:27017/?readPreference=primary&ssl=false';
```

Comment:
`connection-model` does not do this manually now, driver handles this, see changes for the connection-model in https://github.com/mongodb-js/compass/issues/2174